demos/components/README.md

```diff
@@ -0,0 +1,41 @@
+# Chime SDK UI Component Library
+
+This contains reusable componets written with React, Typescript and styled components to be used to create UIs with Chime SDK JS.
+
+## To genereate dependencies
+
+```
+npm install
+```
+
+## To run Storybook locally
+
+```
+npm start
+```
+
+## Build
+
+```
+npm run build
+```
+
+# Usage
+
+After publishing your lib (ie `my-react-lib`) you can use it by importing the styles in your App's entry point:
```

Comment:
Wonder if we could get a small demo app to go along with the PR to demonstrate this with? Maybe that demo could then live in `demos/react` and the `components` folder could live in the root at `/react` or `/components`?

Reply:
I was planning that as a next step. Initially I wanted to land the basic setup for a component library so that others could start utilizing and contributing to it. What are your thoughts?

Reply:
Ok, that makes sense.